product_variant_auto_creation_selectable/README.rst

```diff
@@ -0,0 +1,46 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Product variant automatic creation selectable
+=============================================
+
+This module creates a check in categories and an option in product templates,
+so that it does not create the product variants when the attributes are
+assigned. This is used by those modules that create the product when it is
+strictly needed.
+
+
```

Comment:
It still lacks the Usage section explaining possible values for product categories and templates.